src/network/networkd-routing-policy-rule.c

```diff
@@ -1240,9 +1242,12 @@ int routing_policy_serialize_rules(Set *rules, FILE *f) {
                         space = true;
                 }
 
-                fprintf(f, "%spriority=%"PRIu32,
-                        space ? " " : "",
-                        rule->priority);
+                if (rule->priority != 0) {
```

Comment:
Please use the same conditional as above (`> 0`), having them different looks very strange.

Reply:
Well, I just mimicked the set of `tos`...